src/webots/nodes/utils/WbNodeOperations.hpp

```diff
@@ -73,6 +73,7 @@ public slots:
 signals:
   void nodeAdded(WbNode *node);
   void nodeDeleted(WbNode *node);
+  void fromSupervisorChanged(const bool value);
```

Comment:
This doesn't read very smoothly, I would rename it to:
```suggestion
  void changedFromSupervisor(const bool value);
```